src/components/EventCard/style.js

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+
+export const Card = styled.div`
+border-radius: 8px;
+background-color:#F9F9F9;
+box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.25);
+display: flex;
+flex-direction: column;
+width: 29.6vw;
+height: 22.2vw;
+
+position: absolute;
+left: 100px;
+
+@media screen and (max-width: 1181px) {
+  height: 24vw;
+}
+
+@media screen and (max-width: 944px) {
+  height: 26vw;
+}
+
+@media screen and (max-width: 778px) {
+  height: 28vw;
+}
+
+@media screen and (max-width: 726px) {
+  height: 30vw;
+}
+
+@media screen and (max-width: 609px) {
+  height: 32vw;
+}
+
+@media screen and (max-width: 535px) {
+  height: 34vw;
+}
+`;
```

Comment:
Aqui a gente pode resumir esse Card em:

```suggestion
export const Card = styled.div`
  display: flex;
  flex-direction: column;
  max-width: 24rem;
  width: 100%;
  border-radius: 8px;
  background-color: #f9f9f9;
  box-shadow: 0px 4px 8px #00000025;
  padding: 0.75rem;
  box-sizing: border-box;
`;
```